proto/v1beta1/vulnerability.proto

```diff
@@ -89,6 +89,38 @@ message Vulnerability {
     // obsolete details.
     bool is_obsolete = 9;
   }
+
+  // Windows details get their own format because the information format and model
+  // don't match a normal detail. Specifically windows updates are done as patches,
+  // thus windows vulnerabilities really are a missing package, rather than a package
```

Comment:
nit: capitalize Windows for additional clarity for the reader

Reply:
Applied s/window/Window to the whole cl.